system-tests/ft-suite/src/test/kotlin/com/hotels/styx/admin/ProviderAdminInterfaceSpec.kt

```diff
@@ -119,6 +119,45 @@ class ProviderAdminInterfaceSpec : FeatureSpec() {
                 body shouldContain "/admin/providers/myMonitor/status"
                 body shouldContain "/admin/providers/mySecondMonitor/status"
             }
+
+            scenario("YAML configuration for all providers is available") {
+                val body = styxServer.adminRequest("/admin/providers/objects")
+                        .bodyAs(UTF_8)
+                body shouldContain """
+                    - name: "mySecondMonitor"
+                      type: "HealthCheckMonitor"
+                      tags: []
+                      config:
+                        objects: "bbb"
+                        path: "/healthCheck/y"
+                        timeoutMillis: 250
+                        intervalMillis: 500
+                        healthyThreshold: 3
+                        unhealthyThreshold: 2
```

Comment:
We should add here another object. So that the list would render at least two or more objects.